### PR TITLE
Document that the GenAI plugin is available on prem as well

### PR DESCRIPTION
**Pull request by renetapopova**
Linked to https://github.com/neo4j/docs-operations/pull/1429

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-889.surge.sh

Updated pages:
- [GenAI integrations](https://neo4j-docs-cypher-889.surge.sh/cypher-manual/5/genai-integrations)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-889.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-889.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)
- [Functions](https://neo4j-docs-cypher-889.surge.sh/cypher-manual/5/functions)
- [GenAI integrations](https://neo4j-docs-cypher-889.surge.sh/cypher-manual/5/genai-integrations)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-889.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-889.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)
- [Functions](https://neo4j-docs-cypher-889.surge.sh/cypher-manual/5/functions)
- [GenAI integrations](https://neo4j-docs-cypher-889.surge.sh/cypher-manual/5/genai-integrations)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-889.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-889.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)
- [Functions](https://neo4j-docs-cypher-889.surge.sh/cypher-manual/5/functions)
- [GenAI integrations](https://neo4j-docs-cypher-889.surge.sh/cypher-manual/5/genai-integrations)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-889.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-889.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)
- [Functions](https://neo4j-docs-cypher-889.surge.sh/cypher-manual/5/functions)
- [GenAI integrations](https://neo4j-docs-cypher-889.surge.sh/cypher-manual/5/genai-integrations)

File: modules/ROOT/pages/functions/index.adoc [hunk 1]
```diff
@@ -799,15 +799,14 @@ label:new[Introduced in 5.12]
 |===
 
 [[header-query-functions-genai]]
-**xref::genai-integrations.adoc#single-embedding[GenAI function]** label:new[Introduced in 5.16] label:aura-only[Available only on Aura]
+**xref::genai-integrations.adoc#single-embedding[GenAI function]** label:new[Introduced in 5.17]
```
Comment (nilsceberg) [inline modules/ROOT/pages/functions/index.adoc:802]: No `Available in Aura` label?

File: modules/ROOT/pages/genai-integrations.adoc [hunk 2]
```diff
@@ -2,16 +2,27 @@
...
 
+== Prerequisites
+
+To use the GenAI plugin, you need to have the following:
+
+* A Neo4j DBMS running Neo4j 5.17 or later.
+* A GenAI provider account, such as VertexAI, OpenAI, or Amazon Bedrock.
+* API credentials for the provider you want to use.
+* (On-prem only) the Neo4j GenAI plugin installed in your Neo4j deployment.
+The plugin JAR is located in the _/product_ directory of the Neo4j installation.
```
Comment (nilsceberg) [inline modules/ROOT/pages/genai-integrations.adoc]: This is a bit of a nitpick but is this how usually we format relative paths in the docs? Normally a leading `/` would imply an absolute path. The meaning is easily inferred from the context of course; I'm just double checking. 🙂

**Review by nilsceberg — APPROVED** [review]

File: modules/ROOT/pages/functions/index.adoc (hunk 1 above)
Comment (renetapopova) [reply modules/ROOT/pages/functions/index.adoc:802]: Yes, we don't need the label, as the function and procedure are available in both on-prem and Aura.

File: modules/ROOT/pages/genai-integrations.adoc (hunk 2 above)
Comment (renetapopova) [reply modules/ROOT/pages/genai-integrations.adoc]: Fixed.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-889.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-889.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)
- [Functions](https://neo4j-docs-cypher-889.surge.sh/cypher-manual/5/functions)
- [GenAI integrations](https://neo4j-docs-cypher-889.surge.sh/cypher-manual/5/genai-integrations)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**